std/log/logger.ts

```diff
@@ -37,86 +36,67 @@ export class LogRecord {
   }
 }
 
-export interface LoggerOptions {
-  handlers?: BaseHandler[];
-}
+const DEFAULT_LEVEL = "Debug";
+const DEFAULT_LOGGER_NAME = "logger";
+const defaultConsoleHandler = new ConsoleHandler(DEFAULT_LEVEL);
+const DEFAULT_HANDLERS = [defaultConsoleHandler];
```

Comment:
Please revert the default to `DEFAULT_HANDLERS = []`. Otherwise you can not create a silent logger.

One may want to do this, if one wants to log only if the program is run in certain conditions, e.g. environment flags, etc. One can not just not initialise a logger, because all the logging statements in the code would throw an error.